boards/recovery/src/state_machine/states/landed.rs

```diff
@@ -0,0 +1,31 @@
+use defmt::{write, Format, Formatter};
+
+use crate::no_transition;
+use crate::state_machine::{RocketStates, State, StateMachineContext, TransitionInto};
+use rtic::mutex::Mutex;
+use super::Apogee;
+
+#[derive(Debug, Clone)]
+pub struct Landed {}
+
+impl State for Landed {
+    fn enter(&self,context: &mut StateMachineContext) {
+        context.shared_resources.gpio.lock(|gpio| gpio.fire_main());
+    }
+    fn step(&mut self, _context: &mut StateMachineContext) -> Option<RocketStates> {
+        // Maybe sleep the system here to save power?
```

Comment:
core.scb.sleep or something like that to save power as once we land the recovery board doesn't need to run.